x-pack/plugins/security_solution/public/detections/containers/detection_engine/rules/use_rule_with_fallback.tsx

```diff
@@ -31,6 +32,11 @@ interface AlertHit {
     signal: {
       rule: Rule;
     };
+    kibana: {
```

Comment:
both the `kibana` and `signal` objects in this interface should probably be optional, or this type could be a union of types where one requires `signal` and one requires `kibana` - unless we're building the `AlertHit` objects somehow from the fields response that has both?

Reply:
No, it's one or the either. I updated them to be optional